plan/config_test.go

```diff
@@ -116,3 +133,156 @@ func TestParseBehavior(t *testing.T) {
 		assert.Equal(t, tt.want, parseBehavior(tt.give))
 	}
 }
+
+func TestParseSkipBehavior(t *testing.T) {
+	tests := []struct {
+		give             string
+		desc             string
+		wantFilters      []Filter
+		wantBehaviorName string
+		wantError        error
+	}{
+		{
+			give: "foo=client:c+server:b",
+			desc: "single filter.",
+			wantFilters: []Filter{
+				{
+					AxisMatches: []AxisMatch{
+						AxisMatch{
+							Name:  "client",
+							Value: "c",
+						},
+						AxisMatch{
+							Name:  "server",
+							Value: "b",
+						},
+					},
+				},
+			},
+			wantBehaviorName: "foo",
+			wantError:        nil,
```

Comment:
nit: sounds like the test cases can have either `wantFilters` and `wantBehaviorName` set or just `wantError`. It's okay to leave the parts that you're not testing (`wantError` in this case) to the zero value by not mentioning it in the struct literal.

```
{
  desc: ..,
  give: ..,
  wantFilters: ..,
  wantBehaviorName: ..,
}
```